project/project/CakePlatformDependencies.scala

```diff
@@ -113,7 +113,6 @@ object CakePlatformDependencies {
     val packager: ModuleID =
```

Comment:
playJson is currently only used for `akka-http` and I think janino is mainly used for colouring log output (correct me if I'm wrong).

I will remove the others.